wire-ios-data-model/Source/MLS/SubconversationGroupIDRepository.swift

```diff
@@ -55,7 +55,7 @@ public final class SubconversationGroupIDRepository: SubconversationGroupIDRepos
         _ groupID: MLSGroupID?,
         forType type: SubgroupType,
         parentGroupID: MLSGroupID
-    ) {
+    ) async {
```

Comment:
Why are you annotating the repository functions as async? As understood if you call the actor from the outside they will be async but you don't need to annotate them as as such.

Reply:
you're right